src/Sylius/Bundle/AdminBundle/Resources/config/app/twig_hooks/channel/create.yaml

```diff
@@ -3,11 +3,9 @@ twig_hooks:
         'sylius_admin.channel.create.content':
             form:
                 component: 'sylius_admin:channel:form'
-                data:
-                    channel: '@=resource'
-                    form: '@=form'
-                    parentMainHook: '@=parent_main_hook'
-                    parentFallbackHook: '@=parent_fallback_hook'
+                props:
+                    channel: '@=_context.resource'
```

Comment:
by the way the we will stick out to the resource name rather than simply `resource` ?

Reply:
With templates in mind we could just add both, that's basically in line with how those variables were in the templates before.
With components, it makes meagre sense since it needs to be added as a live prop (probably will be ignored otherwise, though I have not checked), in that case, I'd go w/ the resource name (stuff like `public ?Product $resource = null;` etc. imo looks weird).

Either way we gotta talk it through and decide what's the better option.